rescript

```diff
@@ -105,13 +91,12 @@ function dlog(str) {
     console.log(str);
   }
 }
+
 function notifyClients() {
-  wsClients = wsClients.filter((x) => !x.closed && !x.socket.destroyed);
+  wsClients = wsClients.filter(x => !x.closed && !x.socket.destroyed);
   var wsClientsLen = wsClients.length;
   dlog(`Alive sockets number: ${wsClientsLen}`);
-  var data = JSON.stringify({
-    LAST_SUCCESS_BUILD_STAMP: LAST_SUCCESS_BUILD_STAMP,
-  });
+  var data = '{"LAST_SUCCESS_BUILD_STAMP":' + LAST_SUCCESS_BUILD_STAMP + "}";
```

Comment:
I've decided to inline the JSON. It's 2 times faster https://jsben.ch/TizrB

Reply:
Actually, I didn't really get why we even need the WS. Looks like this is the only message we send, which doesn't look like something super useful. But I've decided to keep everything as it is for now.